src/app/core.ts

```diff
@@ -20,29 +20,30 @@ import { Agent } from 'http';
 import { Credential } from './credential';
 
 /**
- * Available options to pass to [`initializeApp()`](admin#.initializeApp).
+ * Available options to pass to {@link firebase-admin.app#initializeApp}.
  */
 export interface AppOptions {
 
   /**
    * A {@link Credential `Credential`} object used to
    * authenticate the Admin SDK.
    *
-   * See [Initialize the SDK](/docs/admin/setup#initialize_the_sdk) for detailed
-   * documentation and code samples.
+   * See {@link https://firebase.google.com/docs/admin/setup#initialize_the_sdk | Initialize the SDK}
+   * for detailed documentation and code samples.
    */
   credential?: Credential;
 
   /**
-   * The object to use as the [`auth`](/docs/reference/security/database/#auth)
+   * The object to use as the {@link https://firebase.google.com/docs/reference/security/database/#auth | auth}
```

Comment:
Do you know if it's possible to keep the code styling?  

Not required, but nice to have.

Reply:
Haven't been able to get this to work. I've asked tsdoc.org community to see if I've missed something.